CPAC/registration/registration.py

```diff
@@ -4267,10 +4265,8 @@ def warp_deriv_mask_to_T1template(wf, cfg, strat_pool, pipe_num, opt=None):
      "switch": [["registration_workflows", "functional_registration",
                 "func_registration_to_template", "run"],
                 ["registration_workflows", "anatomical_registration", "run"]],
-     "option_key": ["registration_workflows", "functional_registration",
-                    "func_registration_to_template", "apply_transform",
-                    "using"],
-     "option_val": "default",
+     "option_key": "None",
+     "option_val": "None",
```

Comment:
Is this okay? Or should these be applied differently if not

```YAML
registration_workflows:
  functional_registration:
    func_registration_to_template:
      apply_transform:
        using: default
```

?

Reply:
I would leave these as `default` as single-step has its own bold mask warping. We do need it for the derivative-resolution bold mask warping however, which I can add easily in my upcoming PR.

Reply:
196dea240